spec/controllers/reservations_controller_spec.rb

```diff
@@ -1376,80 +1320,61 @@
   end
 
   describe '#renew (PUT /reservations/:id/renew)' do
-    # Access: Admins, checkout persons, users if it's their own reservation
-    # Functionality:
-    # - sets @reservation
-    # - add @reservation.max_renewal_length_available.days to due_date
-    # - add 1 to @reservation.times_renewed
-    # - redirects to @reservation if you can't save
-    # - redirects to root_path if you can save
-    #     (is saving determined by equipment_model.max_renewal_times /
-    #      max_renewal_length?)
-
-    # TODO: Test circumstances under which renewal doesn't/shouldn't work
-
+    # SMELL: this mostly tests permissions
```

Comment:
I'm not really familiar with "SMELL" . Are these todos? Would it be best to leave them out of the final commit?